shardingsphere-db-protocol/shardingsphere-db-protocol-postgresql/src/main/java/org/apache/shardingsphere/db/protocol/postgresql/packet/handshake/PostgreSQLAuthenticationMD5PasswordPacket.java

```diff
@@ -29,6 +29,7 @@
     
     private static final int AUTH_REQ_MD5 = 5;
     
+    @Getter
```

Comment:
Where this member be called?

Reply:
It's called in `PostgreSQLAuthenticationEngineTest`, since `md5Salt` is generated random bytes in `PostgreSQLAuthenticationEngine`, test case will get this `md5Salt` to calculate `md5Digest` for login.

Reply:
Test case will use Java reflection to get it, just like `MySQLAuthenticationEngineTest`.